.github/pull_request_template.md

```diff
@@ -1,78 +1,14 @@
-<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+<!-- Thank you for your contribution! -->
 
+Please provide any of the following information that applies:
```

Comment:
this section needs to be in comments - otherwise it will keep showing up in PRs.